### ISSUE TITLE
Menu doesn't work in Cinnamon 2.4 Linux Mint 17.1

### ISSUE THREAD

**Issue by ghost** [mintystark/starkmenu#7]
Menu doesn't open.

**Comment by mintystark** [comment]
Sorry,
I am aware of this.  I have a fix to make it pop-up, but it is still breaking the quicklinks.
So I need to contact someone that can inform me on what the changes are in 2.4 that I need to do.

**Comment by ghost** [comment]
Hi,
does the shutdown + application menu work on your patched version ?
If yes you sure could at least provide a version with text links on left side  that are defined inside the source code and won`t be changed that esasily just an idea. 
Cause im really a fan of this menu and i do use it every day. Hope that you find a solution that let the amin functions run.

**Comment by ghost** [comment]
typing fault : If yes you sure could at least provide a version with text links -> meaned the right side damn.  am now trying to get cinnamon 2.4 to test it 
regards  bluedxca93

**Comment by mintystark** [comment]
Yeah, the Shutdown still works on the patched version, so I will try and git the Patched version up this weekend.  Then I will look into the Quicklinks issue.

**Comment by ghost** [comment]
now having problems with cinnamon 2.8. it seems as if font color is set to transparent or that text label is not set for quicklinks and all programs.
not trivial error, look at it soon again. configurable menu partially affected too.

**Comment by ghost** [comment]
meaned the all programs/favorites switcher. text is shown if you inspect as widget name but not displayed. normal categorie applications buttons are still working.
